package.json

```diff
@@ -9,7 +9,7 @@
     "migrate": "cd backend && npx sequelize-cli db:migrate",
     "down": "cd backend && npx sequelize-cli db:migrate:undo",
     "seed": "cd backend && npx sequelize-cli seed:generate --name demo-user",
-    "test": "mocha './backend/**/*_test.js' & jest",
+    "test": "jest",
```

Comment:
Why are you removing the back end tests?

Reply:
Thats a woopsie, wanted to just see front end tests, and the seperate front command wasnt in yet